### PR TITLE
Add script to examine statuses from hydra checklist API

### PR DESCRIPTION
**Pull request by wcheang**
Jira: ISV-909

This unfortunately hasn't been tested against the actual Hydra API yet since the account still doesn't have access, but I wanted to request for reviews to get the ball rolling. I'll make sure that it's properly tested before closing out the Jira. I based this on sample output provided by the Hydra team: https://privatebin-it-iso.int.open.paas.redhat.com/?17a0eb36256d1c14#FF7eD1GtG3GvHxiZuhKKTEagEzR87FxAem5JD6gAv4BB

### CODE REVIEW

File: operatorcert/entrypoints/hydra_checklist.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    # if checklist is not complete, go through each item and log which ones are missing
+    checklist_items = hydra_resp["checklistItems"]
+    completed = True
+    for item in checklist_items:
+        completed = completed and item["completed"]
+        if not item["completed"]:
+            LOGGER.info(f"Checklist item not completed: {item['title']}")
+
+    if not completed:
+        LOGGER.info(
```
Comment (MarcinGinszt) [inline operatorcert/entrypoints/hydra_checklist.py:58]: Nitpick: I think this could be LOGGER.error

**Comment by MarcinGinszt** [comment]
Code looks good, waiting for Hydra access to verify! :)

File: operatorcert/entrypoints/hydra_checklist.py (hunk 1 above)
Comment (wcheang) [reply operatorcert/entrypoints/hydra_checklist.py:58]: Updated

**Comment by wcheang** [comment]
Thanks for the review Marcin! I got the correct password for the other Hydra account, and was able to test the script. The output aligns with the sample one I got from the Hydra team before, so there were no changes needed. Merging this now.

**Pull request merged**